tools/create-openbsd-gce-ci.sh

```diff
@@ -80,15 +80,13 @@ EOF
 cat >etc/vm.conf <<EOF
 vm "syzkaller" {
   disable
-  disk "/syzkaller/syzkaller.img"
+  disk "/syzkaller/userspace/image"
```

Comment:
This image will not be actually used, right? We don't want to use this one, we have a per-VM private copy of the image. If it's not used, do we need to specify it?

Reply:
Right, but I can't omit the disk line. vmd complains on startup:
```
Sep 27 10:18:38 ci-openbsd vmd[16182]: /etc/vm.conf:6: vm "syzkaller" failed: Invalid argument
```

Even though vmd -n is happy with the missing disk line:
```
ci-openbsd# vmd -n                                                                                                     
configuration OK                                                                                                       
```

I ended up using `/dev/null`.

Incidentally, this was also the reason for vmd failing to start for me. It was really confusing due to vmd -n checking for disk file presence on `/syzkaller` fs. This fs is mounted _after_ `/etc/rc` tried to run `rcctl start vmd` and displayed an uninformative `(failed)`. Later on rc.local mounts `/syzkaller` and I can't reproduce the problem. Now `rcctl start vmd` just happily sees the image. Anyway, I think is resolved now.